### DISCUSSION TITLE
multiple companies with only installation

### DISCUSSION THREAD

**Discussion by stefano-bertaiola** [globaleaks/globaleaks-whistleblowing-software#3452] [Q&A]
HI
I have multiple companies, can I use only installation using channels as a company?
Thank you

**Comment by evilaliv3** [comment]
Thank you Stefano for your question.

This is definitely doable.

Globaleaks includes a features called channels that be used for:
- creating multiple channels per topic for the same company
- creating multiple channels on the same company for controlled companies
(some legal firms use this feature to handle their own client companies)

Than there is a Multisite feature that can be used to create multiple
independent tenants. For example our demo system (try.globaleaks.org) is a
multitenant setup just based on GlobaLeaks and as well the project
WhistleblowingPA (www.whistleblowing.it) is based on this.
